http/src/lib.rs

```diff
@@ -20,7 +20,7 @@
 #![warn(missing_docs)]
 
 extern crate unicase;
-extern crate jsonrpc_core as jsonrpc;
+pub extern crate jsonrpc_core;
```

Comment:
Would prefer `pub extern` in a separate section

Reply:
Yeah, I will put below near `hyper`. Not sure why I did it like that.